Documentation/Testing/ExtensionTesting.rst

```diff
@@ -132,7 +124,7 @@ This is how the composer.json file looks before we add a test setup:
         "GPL-2.0-or-later"
       ],
       "require": {
-        "typo3/cms-core": "^9.5"
+        "typo3/cms-core": "^11.5"
```

Comment:
I guess we should add some note somewhere below, hinting at least about the issue with using `typo3/cms-composer-installers 4RC1` along with TYPO3 v11, special for using Testing Framework later on (needs my bridge plugin for TF6 -> https://github.com/sbuerk/typo3-cmscomposerinstallers-testingframework-bridge).

With TF7+ the bridge plugin is obsolete and TF is cabable of handling different places for ext/sysexts. Which needs other entry points then. But at least, we should add a "hint" to that issue somehow.

If I was not clear enough or could not transport what I ment, ping me in slack.

Reply:
That is pretty much an edge case (as discussed) but it was pointed out not uncommon, so I will add the note.

Reply:
Can we handle the testingframework bridge in another patch please? As I wrote [elsewhere](https://github.com/sbuerk/typo3-cmscomposerinstallers-testingframework-bridge/issues/11) I have not fully understood when it is required. It would be better if someone else takes care of this. 

Also, it would be easier with the merging and up-porting (adapted for v12 and main) if we do not put too much into individual patches.